packages/hooks/src/useCryptoConfig.ts

```diff
@@ -0,0 +1,15 @@
+import { useQuery } from '@deriv/api';
+
+const useCryptoConfig = (currency: string) => {
+    const { data, ...rest } = useQuery('crypto_config');
```

Comment:
```suggestion
    const { data, ...rest } = useQuery('crypto_config');
```
@heorhi-deriv, isn't it possible to directly pass the `currency` as part of the `payload` to the endpoint here?

Reply:
yes @jim-deriv, make sense
let me try to refactor it

Reply:
sorted